src/main/kotlin/org/ionproject/integration/model/internal/Response.kt

```diff
@@ -0,0 +1,21 @@
+package org.ionproject.integration.model.internal
+
+data class Response(val statusCode: Int, val body: ByteArray) {
+    override fun equals(other: Any?): Boolean {
```

Comment:
Why not use a data class here?

Reply:
Oh, this is a data class, sorry. So why the need to change `equals` and `hashCode`? Please add the reasoning as a comment to this class.